### PR TITLE
fix(callout-quote): content off grid

### LINKED ISSUE #10050: [Callout quote]: Content off grid
### Description
The `callout quote` component's quote text, attribution, and link content are not aligned in the live version. See the [design specs](https://www.figma.com/file/KQdUduecrBf4DCjRZnsl25/Layout-component--callout-quote--visual-specs--2.0.0?node-id=0%3A1&t=rY2GJtnjEA2Mv9U5-1) for proper grid alignment.

Seeing the misalignment from `@max` through `@sm`.

**@xlg**
Screen Shot 2023-02-14 at 11 14 59 AM

**@md**
Screen Shot 2023-02-14 at 11 15 21 AM

**@sm**
Screen Shot 2023-02-14 at 11 16 40 AM

**Compare with [Figma design specs](https://www.figma.com/file/KQdUduecrBf4DCjRZnsl25/Layout-component--callout-quote--visual-specs--2.0.0?node-id=0%3A1&t=rY2GJtnjEA2Mv9U5-1)**
Screen Shot 2023-02-14 at 11 19 57 AM
Screen Shot 2023-02-14 at 11 22 26 AM

### Component(s) impacted

`callout quote`

### Browser

Chrome, Safari, Firefox

### Carbon for IBM.com version

v1.28.0

### Severity

Severity 3 = The problem is visible or noticeable to users but does not impede the usability or functionality. Affects minor functionality, has a workaround.

### Application/website

C4IBM.com

### Package

@carbon/ibmdotcom-react, @carbon/ibmdotcom-web-components, @carbon/web-components, @carbon/ibmdotcom-styles

### CodeSandbox example

https://codesandbox.io/s/github/carbon-design-system/carbon-for-ibm-dotcom/tree/main/packages/web-components/examples/codesandbox/components/callout-quote

### Steps to reproduce the issue (if applicable)

_No response_

### Release date (if applicable)

_No response_

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

### PR DESCRIPTION
**Pull request by sangeethababu9223**
### Related Ticket(s)

Closes #10050

### Description

Callout Grid - Content was Off Grid.

### Changelog

**New**

- Added an offset to match the figma
- adjusted padding

**Changed**

- Changed Column width for the grid.

### CODE REVIEW

**Comment by sangeethababu9223** [comment]
I have read the DCO document and I hereby sign the DCO.

**Comment by sangeethababu9223** [comment]
Related Pull request: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/10865

**Comment by sangeethababu9223** [comment]
recheck

**Review by RichKummer — COMMENTED** [review]
Hey @sangeethababu9223 , it looks like we're getting there! The breakpoionts `@sm` and `@md` look aligned with the grid. I am seeing that the desktop sizes are slightly off. You can see the slight difference below:

**@lg**
Screenshot 2023-08-24 at 10 52 05 AM

**@xlg**
Screenshot 2023-08-24 at 10 52 20 AM

**@max**
Screenshot 2023-08-24 at 10 52 31 AM

**Comment by sangeethababu9223** [comment]
Hi @RichKummer ,
I've adjusted the alignment. Could you please check once again.

**Review by RichKummer — APPROVED** [review]
This is amazing! Thanks for sticking with this @sangeethababu9223 🎉 🎉 🎉

File: packages/web-components/src/components/callout-quote/callout-quote.scss [hunk 1]
```diff
@@ -33,3 +33,18 @@
...
   }
 }
+
+:host(#{$dds-prefix}-callout-quote) {
+  .#{$prefix}--quote__wrapper {
+    margin-left: -1rem;
+    margin-right: -1rem;
+    @include carbon--breakpoint('md') {
+      margin-left: -1rem;
+      margin-right: -1rem;
```
Comment (emyarod) [inline packages/web-components/src/components/callout-quote/callout-quote.scss:43]:
```suggestion
    margin-left: -$spacing-05;
    margin-right: -$spacing-05;
    @include carbon--breakpoint('md') {
      margin-left: -$spacing-05;
      margin-right: -$spacing-05;
```

**Review by emyarod — COMMENTED** [review]
looks good to me! just one thing, I believe we should prefer using [Carbon spacing tokens](https://carbondesignsystem.com/guidelines/spacing/overview/) wherever we can

File: packages/web-components/src/components/callout-quote/callout-quote.scss (hunk 1 above)
Comment (sangeethababu9223) [reply packages/web-components/src/components/callout-quote/callout-quote.scss]: Thanks @emyarod .
I've updated the style with spacing token.

**Pull request merged**